main.py

```diff
@@ -4,6 +4,9 @@
 from colorama import init as colorama_init, Fore, Style
 from terminal_gui import user_input, select_input
 from core.lookup import web_lookup
+import os
+
+os.environ['KMP_DUPLICATE_LIB_OK']='True'
```

Comment:
please do not push this to the main codebase, this is a workaround for only some mac versions